extensions/pkg/terraformer/state.go

```diff
@@ -160,3 +163,45 @@ func sniffJSONStateVersion(stateConfigMap []byte) (uint64, error) {
 
 	return *sniff.Version, nil
 }
+
+// Initialize implements
+func (f StateConfigMapInitializerFunc) Initialize(ctx context.Context, c client.Client, namespace, name string) error {
+	return f(ctx, c, namespace, name)
+}
+
+// CreateState create terraform state config map and use empty state
+// It does not create or update state ConfigMap if alredy exists
+func CreateState(ctx context.Context, c client.Client, namespace, name string) error {
```

Comment:
cc @ialidzhikov - can you double-check these changes to the `terraformer` pkg here?